core/src/main/kotlin/org/evomaster/core/problem/rest/resource/RestResourceCalls.kt

```diff
@@ -184,8 +184,8 @@ class RestResourceCalls(
      */
     fun seeActions(filter: ActionFilter): List<out Action> {
         return when (filter) {
-            ActionFilter.ALL -> sqlActions.plus(externalServiceActions).plus(mainActions)
-            ActionFilter.INIT -> sqlActions.plus(mongoDbActions)
+            ActionFilter.ALL -> sqlActions.plus(externalServiceActions).plus(mainActions) // FIXME: Is this correct?
```

Comment:
once the refactoring of `EnterpriseIndividual` is completed, i ll refactor all those action filters. so, no need to worry too much about this now